docs/projects/website.md

```diff
@@ -118,6 +118,12 @@ Changes to the front-end project must be reverted from your Git client.
 
 Using a RichText JSS element with a `tag="p"` prop causes issues when editing the element in Horizon. To be specific, Horizon will wrap your plain text inside a paragraph which results in nested `p` tags and an invalid HTML. For this reason it is suggested to use a block element that has no children restrictions like `div`, `section`, `blockquote`, etc.
 
+#### Custom Renderings Icons
```

Comment:
For us to add more icons in the future, could you please:
1. Commit the icons ZIP file content into `C:\projects\Sitecore.Demo.Edge\Website\src\icons`
2. Document how to:
   1. Add icons (where to put the files, what format, what size, etc.)
   2. Generate a new ZIP file.
   3. Test new icons using the quick development feedback loop (copying to /deploy/... folder).
   4. Override the existing zip file in the /docker folder before committing.

Thank you very much

Reply:
I realize that I did not add enough details to that first request about committing the individual icons. I should have specified to add only the icons we are using in our renderings.

Can the unused icons please be removed from source control and from the ZIP file in the docker folder?

Reply:
The `announcement.png` icon seems to be missing in all 4 folders and the zip file.